js/src/site.js

```diff
@@ -14,6 +14,16 @@
 			scrollTop = 0,
 			windowHeight = 0,
 			windowWidth = 0,
+			min_xs = "@mq_min_xs",
```

Comment:
Good catch, @timbuckingham, I'm updating this. I'm not completely sold on the naming and am all for making them more readable. The reason I was going with xs, sm, md, lg, xl rather than key names is because of the scale. If we need to need a larger breakpoint we could add xxl, obviously larger. If it's "huge" and we need a larger ones, I fear it would be less clear. Surely, something like "extreme" would be larger than "huge" but maybe not compared to "gigantic" or similar.